src/en/clouds.md

```diff
@@ -116,7 +116,7 @@ There are three special types of clouds: MAAS, LXD and Manual.
   developing your own Juju charms - it is incredibly fast! 
   [LXD is a container hypervisor][LXD-site] that runs on any Linux host, providing 
   the ability to spin up containers on the host machine. For more details on
-  using LXD, please see the [LXD documentation][juju-lxd].
+  using LXD, please see the [Using LXD with Juju][clouds-lxd].
```

Comment:
needs *page* or *documentation* before the end stop.

Reply:
:astonished: